glacier/src/test/importer.spec.ts

```diff
@@ -5,6 +5,8 @@ import {DOMParser} from "xmldom";
 import {evaluate, XPathResult} from "xpath";
 import {Store} from "redux";
 import * as fs from "fs";
+import {resolve} from "path";
+const jszip = require("jszip");
```

Comment:
It aught to. If it doesn;t you may need to add jzip/the jzip types to the secondary package.json (one level up from the one in src?)